frontend/src/features/profile/components/PersonalAccountDetails.tsx

```diff
@@ -0,0 +1,20 @@
+import type { IUser } from "../../../../../shared/user.interface";
+
+interface PersonalAccountDetailsProps {
+  user: IUser;
+}
+
```

Comment:
as we have now i18n added, please extract the strings(texts) and use i18n for the supported languages.
frontend\src\messages

for reference of usage:
frontend\src\features\landing-page\components\Introduction.tsx
  const t = useTranslations("landing.introduction");